storm-server/src/test/java/org/apache/storm/scheduler/resource/strategies/scheduling/TestDefaultResourceAwareStrategy.java

```diff
@@ -905,9 +911,9 @@ public void testMultipleRacksWithFavoritism() {
 
         rs.prepareForScheduling(cluster, topo1);
         INodeSorter nodeSorter = new NodeSorter(cluster, topo1, BaseResourceAwareStrategy.NodeSortType.DEFAULT_RAS);
```

Comment:
Do we want to change this to NodeSorterHostProximity too?

or why don't we just rename `NodeSorterHostProximity` to `NodeSorter` and remove the current NodeSorter class?

Reply:
Yes. Changed to use NodeSorterHostProximity.

Rest of the code cleanup of unused classes (after some discussion on NodeSortType) will be in https://issues.apache.org/jira/browse/STORM-3747